src/read/abbrev.rs

```diff
@@ -281,6 +280,102 @@ impl Abbreviation {
     }
 }
 
+/// A list of attributes found in an `Abbreviation`
+#[derive(Clone)]
+pub struct Attributes {
```

Comment:
Let's make this `pub(crate)` instead, and change `Abbrevation::new` to be `pub(crate)` too, and same for `AbbreviationSpecification::new` while we're at it.